mojo/services/keyboard/interfaces/BUILD.gn

```diff
@@ -9,4 +9,10 @@ mojom("interfaces") {
   sources = [
     "keyboard.mojom",
   ]
+
+  import_dirs = [ get_path_info("../../", "abspath") ]
+
+  deps = [
+    "../../native_viewport/interfaces",
```

Comment:
Doing a quick survey on mojo/services, it seems like most places use "../../" to refer to mojo/services, and a source root path to refer to mojo/public/interfaces (which doesn't have what I need - I'm not sure what the distinction is).